examples/java-basic-mvn/src/main/java/pineconeexamples/UpsertsAndQueriesConcurrentExample.java

```diff
@@ -1,46 +1,36 @@
 package pineconeexamples;
 
-import io.pinecone.PineconeClient;
-import io.pinecone.PineconeClientConfig;
-import io.pinecone.PineconeConnection;
-import io.pinecone.PineconeConnectionConfig;
-import io.pinecone.PineconeException;
-import io.pinecone.PineconeResponse;
-import io.pinecone.QueryResponse;
-import io.pinecone.UpsertResponse;
+import com.google.common.primitives.Floats;
+import io.pinecone.*;
+import io.pinecone.proto.*;
```

Comment:
Could you expand these so that readers see exactly where symbols like UpsertResponse are coming from below?